### PR TITLE
implement get symptoms api v2

### PR DESCRIPTION
**Pull request by nodestory**
Since the response format already changes, so v2 API is created.

- There are three types of symptoms now: `official`, `suggested` and `customized`.
- API changes:
`GET /v2/symptoms` returns official and customized symptoms.
`GET /v2/symptoms?all=true` returns all available symptoms for the client side to build auto-completion list. The language setting only affects official and suggested symptoms.

The actual suggested symptoms are still under review and will be added after this PR.

### CODE REVIEW

File: schema/command/migrate/main.go [hunk 1]
```diff
@@ -227,16 +222,32 @@ func setupCollectionSymptom(ctx context.Context, client *mongo.Client) error {
...
+	officialSymptoms := make([]interface{}, 0, len(schema.COVID19Symptoms))
+	for _, s := range schema.COVID19Symptoms {
+		officialSymptoms = append(officialSymptoms, s)
+	}
+	_, err := c.InsertMany(ctx, officialSymptoms)
 	if err != nil {
 		if errs, hasErr := err.(mongo.BulkWriteException); hasErr {
-			if 1 == len(errs.WriteErrors) && store.DuplicateKeyCode == errs.WriteErrors[0].Code {
-				return nil
+			if !(1 == len(errs.WriteErrors) && store.DuplicateKeyCode == errs.WriteErrors[0].Code) {
```
Comment (pieceofr) [inline schema/command/migrate/main.go:232]: && --> ||

File: store/symptom.go [hunk 2]
```diff
@@ -111,6 +114,74 @@ func (m *mongoDB) ListOfficialSymptoms(lang string) ([]schema.Symptom, error) {
...
+			return nil, err
+		}
+
+		if name, err := loc.Localize(&i18n.LocalizeConfig{
+			MessageID: fmt.Sprintf("symptoms.%s.name", s.ID),
+		}); err == nil {
+			s.Name = name
+		} else {
+			log.WithError(err).Warnf("can not decode name")
+		}
```
Comment (lemonlatte) [inline store/symptom.go:152]: Do we translate suggested symptoms? If not, we should remove this part.

Comment (lemonlatte) [reply store/symptom.go:152]: If we do translate those symptoms, we need to consider how to translate them in customized symptoms as well.

Comment (nodestory) [reply store/symptom.go:152]: We also provide translations for suggested symptoms.

Comment (nodestory) [reply store/symptom.go:152]: After the discussion with Casey, we decided we only translate official and suggested symptoms for now.

**Pull request merged**